package.json

```diff
@@ -34,7 +34,10 @@
     "benchmark": "benchr test/benchmark.js",
     "test": "mocha test/*.test.js",
     "doctoc": "doctoc --notitle README.md",
-    "profile": "bin/nearleyc.js test/grammars/parens.ne > test/grammars/parens.js && node test/profile.js"
+    "profile": "bin/nearleyc.js test/grammars/parens.ne > test/grammars/parens.js && node test/profile.js",
+    "fix": "npm run eslintfix; npm run prettier",
+    "eslintfix": "eslint --fix lib/; eslint --fix test/; eslint --fix bin/",
+    "prettier": "prettier --tab-width 4 --write lib/**/*.js test/**/*.js bin/**/*.js"
```

Comment:
use "list-different" option to fail the CI if the formatting deviated from the standard
of prettier.

See: https://github.com/prettier/prettier#--list-different